packages/components/label/src/stories/Styled.vue

```diff
@@ -3,7 +3,7 @@ import { OkuLabel } from '@oku-ui/label'
 </script>
 
 <template>
-  <OkuLabel class="label">
+  <OkuLabel class="label" @click="console.log">
```

Comment:
I don't think there's a need for console.log here; the event is working, and the text is not selectable when double-clicking on the label.